app/jobs/socure_docv_results_job.rb

```diff
@@ -42,14 +42,62 @@ def perform(document_capture_session_uuid:, async: true, docv_transaction_token_
           passport_image_fingerprint: nil,
           selfie_image_fingerprint: nil,
         )
+
+        record_attempt(docv_result_response:, doc_pii_response:)
         return
       end
     end
+
+    record_attempt(docv_result_response:, doc_pii_response:)
     document_capture_session.store_result_from_response(docv_result_response)
   end
 
   private
 
+  def record_attempt(docv_result_response:, doc_pii_response: nil)
+    image_errors = {}
+
+    if doc_escrow_enabled?
```

Comment:
this should probably include `if sp&.attempts_api_enabled?` -- otherwise we'll be pinging the endpoint for all integrations when doc_escrow is enabled generally

Reply:
Agree, we should check that it's both enabled generally and for the specific SP.